src/sentry/static/sentry/app/utils/apm.jsx

```diff
@@ -66,13 +61,11 @@ export function finishTransaction(delay = TRANSACTION_TIMEOUT) {
   }
 
   flushTransactionTimeout = setTimeout(() => {
-    Sentry.configureScope(scope => {
-      const span = scope.getSpan();
-      if (span) {
-        span.finish();
-        firstPageLoad = false;
-      }
-    });
+    if (currentTransactionSpan) {
```

Comment:
@dashed Changed this up a bit too, we should be specific about the span to be closed.

Reply:
@billyvg 👍 👍 👍